reinforcement_learning/rlclientlib/live_model.cc

```diff
@@ -43,34 +44,34 @@ namespace reinforcement_learning
                               api_status* status)
   {
     INIT_CHECK();
-    return _pimpl->choose_rank(event_id, context_json, response, status);
+    return choose_rank(event_id, context_json, action_flags::DEFAULT, response, status);
   }
 
   int live_model::choose_rank(const char* context_json, ranking_response& response, api_status* status)
   {
     INIT_CHECK();
-    return _pimpl->choose_rank(context_json, response, status);
+    return choose_rank(context_json, action_flags::DEFAULT, response, status);
   }
 
   //not implemented yet
```

Comment:
[Minor] Can be removed

Reply:
you mean both methods with legacy signature (without flags)?

Reply:
The comment about "not implemented yet" is superfluous in both.